### PR TITLE
[US118 Task146] Add Wiki Page and Improve Dashboard

### PR DESCRIPTION
**Pull request by AdvaitChirmule**
Screenshot:

image

### CODE REVIEW

File: react-ui/src/metricwiki/metricwiki.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                            <h4><u>Lead Time</u></h4>
+                            Lead Time is defined as the time duration from the creation of the process to its completion. Lead Time 
+                            is an important metric because it helps a developer team estimate the time required to deliver a 
+                            requirement defined by the customer.<br/>
+                            <br/>
+                            <b>Input:</b> <br/>
+                            The Project Slug, and the Date Range between which the Lead Time is to be measured. <br/><br/>
+                            <b>Output:</b> <br/>
+                            (i) A Scatter Graph representing the Lead Times of user stories within the specified date range. <br/>
+                            (i) A Scatter Graph representing the Lead Times of individual tasks within the specified date range. <br/>
```
Comment (spark290) [inline react-ui/src/metricwiki/metricwiki.js]: Change (i) to (ii)

File: react-ui/src/metricwiki/metricwiki.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                        <div id="CycleTimeInfo">
+                            <h4><u>Cycle Time</u></h4>
+                            Cycle Time is defined as the time duration from the start of the process to its completion. Cycle Time is 
+                            an important metric because it helps a developer team estimate the time required to work on a product.<br/>
+                            <br/>
+                            <b>Input:</b> <br/>
+                            The Project Slug, and the Date Range between which the Cycle Time is to be measured. <br/><br/>
+                            <b>Output:</b> <br/>
+                            (i) A Scatter Graph representing the Lead Times of user stories within the specified date range. <br/>
+                            (i) A Scatter Graph representing the Lead Times of individual tasks within the specified date range. <br/>
```
Comment (spark290) [inline react-ui/src/metricwiki/metricwiki.js]: Change (i) to (ii)

File: react-ui/src/aboutus/aboutus.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                            </div>
+                        </Navbar>
+                    </div>
+                    <div className="backgroundWhite" style={{ minHeight: "85vh", padding: "30px" }}>
+                        <Container>
+                            <Row>
+                                <Col sm={8}>
+                                    <h2>Meet the Developer Team!</h2><br />
+                                    We are a developer team of five members: Ashutosh Chaurasia (achaur11@asu.edu), Advait Chirmule (achirmul@asu.edu), Hasan Shahid (hshahid@asu.edu),
+                                    Sehun Park (spark29@asu.edu) and Shikha Verma (sverma27@asu.edu).
```
Comment (spark290) [inline react-ui/src/aboutus/aboutus.js]: Change misspelling (spark29 to spark290)

File: react-ui/src/metricwiki/metricwiki.js (hunk 2 above)
Comment (AdvaitChirmule) [reply react-ui/src/metricwiki/metricwiki.js]: Done

File: react-ui/src/aboutus/aboutus.js (hunk 3 above)
Comment (AdvaitChirmule) [reply react-ui/src/aboutus/aboutus.js]: Fixed

File: react-ui/src/metricwiki/metricwiki.js (hunk 1 above)
Comment (AdvaitChirmule) [reply react-ui/src/metricwiki/metricwiki.js]: Done

**Review by spark290 — APPROVED** [review]
LGTM

**Pull request merged**